frontend/src/main/scala/bloop/engine/Build.scala

```diff
@@ -1,13 +1,27 @@
 package bloop.engine
 
 import bloop.Project
-import bloop.io.AbsolutePath
+import bloop.io.{AbsolutePath, DirChecksum}
+
+final case class Build private (origin: AbsolutePath,
+                                projects: List[Project],
+                                originChecksum: DirChecksum) {
```

Comment:
`DirChecksum` is not a case class, contrary to what `Build` is. That means that comparing builds that may represent the same build will always yield false (because of object equality in `DirChecksum`). Should we make `DirChecksum` a case class too? What are the disadvantages of doing that?

Reply:
Making `DirChecksum` a case class sounds good.